engines/native/resultset.go

```diff
@@ -140,17 +140,21 @@ func (r *resultSet) ExtractFolder(path string, handler engines.FileHandler) erro
 }
 
 func (r *resultSet) Dispose() error {
-	// Halt all other sub-processes
-	err := system.KillByOwner(r.user)
-	if err != nil {
-		r.log.Error("Failed to kill all processes by owner, error: ", err)
-	}
+	var err error
+
+	if r.engine.config.CreateUser {
```

Comment:
Hmmm... I guess we can't, am I right? Because the process is already dead at this point, so we can't do further attempts to cleanup... Children were either killed or now have parent id 1.